<launch>
  <node name="jint" pkg="project4_intpol" type="jint.py" output="screen" />
  <node name="oint" pkg="project4_intpol" type="oint.py" output="screen" />
  <node name="kdl_dkin" pkg="project3_dkin" type="kdl_dkin.py" output="screen" />
</launch>
